openquake/hazardlib/contexts.py

```diff
@@ -43,8 +43,8 @@
 bymag = operator.attrgetter('mag')
 bydist = operator.attrgetter('dist')
 I16 = numpy.int16
-KNOWN_DISTANCES = frozenset(
-    'rrup rx ry0 rjb rhypo repi rcdpp azimuth azimuth_cp rvolc'.split())
+tmp = 'rrup rx ry0 rjb rhypo repi rcdpp azimuth azimuth_cp rvolc closest_pnts'
```

Comment:
Please, let call it `closest_points` to be consistent with `rupture.surface.get_closest_points`.

Reply:
I called it `closest_point` since for each site we effectively find one point on the rupture